Actors/VisualObjects/VisualObjects.WebService/App.config

```diff
@@ -1,18 +1,18 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <startup>
-    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.1" />
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
```

Comment:
I don't think we can target 4.5.2 yet since last we knew this was not the default .net version on azure vm images. Can you confirm one way of the other then change these back to 4.5.1?